### DISCUSSION TITLE
My Account has been flagged

### DISCUSSION THREAD

**Discussion by ytgweb3ninja** [community/community#46927] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Product Feedback

### Body

Hi Team, My account has been flagged. Can you please revert back?

**Comment by queenofcorgis** [comment, marked as answer]
Hi @ytgweb3ninja ! The best place to go for answers to account-specific questions like this one is opening a ticket on our [Support page](https://support.github.com/contact). Please open a ticket with them as the community is unable to assist so I'll be locking this post.  :)

**Reply by hamidmehdizadeh** [reply]
Hi @queenofcorgis ,
I have the same problem. I have sent a ticket to the support page, but even that one is missing! What should I do? Issue another ticket? Thanks for your help beforehand.

**Reply by kaichain** [reply]
they answer too slow. We also have this issue and still waiting for their reply. We want to upgrade to at least PRO but cannot upgrade due to this issue.

**Reply by Zibri** [reply]
6 hours.. still waiting after opening a ticket.. I don't even know why it was flagged.

**Comment by sutiliza** [comment]
Hello!  could I re-enable my account's public visibility?

**Comment by sutiliza** [comment]
after these years, I can better understand how github works

**Comment by Avneesh-chaurasiya** [comment]
Hi Team, My account has been flagged. Can you please revert back?

**Comment by tegs1** [comment]
Hi Team, My account has been flagged. Can you please revert back?

**Comment by jcaspdm** [comment]
Hello Team, My account has been flagged. Can you please revert back?

**Comment by rintisch** [comment]
Hello team,
my account has been flagged. I opened a support ticket but I got still no answer. :(
Can you please revert back?

**Comment by Lukabrazi111** [comment]
Hi Git support team, my account got hacked and someone add too many repositories in my account with *Cheats* and hack (games),
I removed all repositories and reset my password but my account has been flagged can you help me please?

**Reply by Lukabrazi111** [reply]
@queenofcorgis can you help me please?

**Reply by YossefMohamed** [reply]
any updates about your account ?

**Comment by TanvirJahan001** [comment]
Hi Team, My account has been flagged. Can you please revert back?

**Comment by YossefMohamed** [comment]
Hi Team, My account has been flagged. Can you please revert back?
